setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# black compatibiliti
+# https://github.com/psf/black/blob/main/docs/guides/using_black_with_other_tools.md#flake8
+max-line-length = 120
```

Comment:
`black` will use 88 though, should be in line/sync.

Reply:
`--max-line-length=120` specified in `make lint`
I just put this parameter in `setup.cfg`, so VS Code won't complain about strings longer than 80 characters (flake8)

Reply:
Although I agree that they should be in sync generally, there's like 50+ docstrings that'd need to be fixed. I'm keen to get this merged in for now and then we can revisit it.